pkgs/misc/cups/drivers/hl3140cw/default.nix

```diff
@@ -0,0 +1,76 @@
+{stdenv, fetchurl, cups, dpkg, gnused, makeWrapper, ghostscript, file, a2ps, coreutils, gawk}:
+
+let
+  version = "1.1.4-0";
+cupsdeb = fetchurl {
+  url = "https://download.brother.com/welcome/dlf007070/hl3140cwcupswrapper-${version}.i386.deb";
+  sha256 = "a76281828ca6ee86c63034673577fadcf5f24e8ed003213bdbb6bf47a7aced6f";
+};
+srcdir = "hl3140cw_cupswrapper_GPL_source_${version}";
+cupssrc = fetchurl {
+  url = "https://download.brother.com/welcome/dlf006740/${srcdir}.tar.gz";
+  sha256 = "1wp85rbvbar6rqqkaffymxjpls6jx9m9230dlrpqwy5akiaxf0rl";
+};
+lprdeb = fetchurl {
+  url = "https://support.brother.com/g/b/files/dlf/dlf007068/hl3140cwlpr-1.1.2-1.i386.deb";
+  sha256 = "601f392b52ed7080f71b780181823bb8f6abfd0591146b452ba1f23e21f9f865";
+};
+  in
+stdenv.mkDerivation {
+  name = "cups-brother-hl3140cw";
+
+  srcs = [ lprdeb cupssrc cupsdeb ];
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ cups ghostscript dpkg a2ps ];
```

Comment:
`dpkg` seems to be used to unpack the source, so I think it should be in `nativeBuildInputs`